CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.2.0dev - [date]
+## v2.2.0 - [date]
```

Comment:
@mirpedrol do you know why this is changed by the template update?

Reply:
we can continue with our version and not accept the template change :)